CODEOWNERS

```diff
@@ -0,0 +1,14 @@
+*  @fjetter
+
+# Dashboard
+distributed/dashboard/*  @jacobtomlinson
+distributed/diagnostics/*  @jacobtomlinson @fjetter
```

Comment:
How does this work exactly? I am the "default" owner. you are setting yourself as "dashboard" owner. Will I then not get any pings for this? i.e. is it `default and specific` or `specific or default`?

Reply:
ok, it is `OR`

this is discussed in their example file https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file

Reply:
I *think* more specific matches overrule the less specific ones.

> I am the "default" owner. you are setting yourself as "dashboard" owner. Will I then not get any pings for this?

I *think* so. Any dashboard changes will explicitly ask for my review, but not yours. Your normal notifications will be unchanged though (I assume you subscribe to all notifications). `CODEOWNERS` just automatically requests reviews.

Reply:
> I assume you subscribe to all notifications

Yes, I'm mostly asking to clarify my understanding. Nothing here is binding anyhow so we can iterate if something is missing/too much

Reply:
I'm partly just experimenting here and we can see how it behaves in practice and update it if this doesn't make sense. Its interesting to look at the codebase through an "ownership lens" because the files are not always partitioned neatly that way (or not at all, `scheduler.py` is huuuuuge).

Reply:
>  (or not at all, scheduler.py is huuuuuge).
🪓 ✂️